### DISCUSSION TITLE
Qual a maneira correta de adicionar uma permissão para uma nova funcionalidade ?

### DISCUSSION THREAD

**Discussion by marquimConfig** [RamonSilva20/mapos#1783] [Q&A]
Opa, vou tentar explicar melhor.

Estou estudando o código, pois achei muito interessante e util. E imaginei uma possibilidade de adicionar uma necessidade que preciso no sistema, então comecei a procurar como fazer, modifiquei algumas coisas, mas o que não consegui alterar, nem adicionar, foi a questão de permissões para gerir qual usuário teria o acesso.

P-> O que eu preciso ? 
R-> Adicionar determinada rotas para a O.S. Nisso irei criar uma área de rotas (Cadastro, Edição, e Exclusão), e adicionar uma lista dessas rotas no cadastro e na edição para a O.S, e no final, adicionar um filtro de rotas para puxar os relatórios de O.S.

Essa parte eu já tenho ideia de como fazer. Porem pra começar, preciso definir as permissões de acesso. Algo que estou batendo cabeça adicionar, mas sem sucesso até o momento.

Qualquer ajuda, serei grato. Desde já agradeço a atenção !

**Comment by Pr3d4dor** [comment]
@marquimConfig Não consegui entender exatamente a sua dúvida.

**Reply by marquimConfig** [reply]
No caso, eu queria adicionar mais uma opção de permissão. Que seria para visualizar adicionar editar e deletar as rotas.
